index.js

```diff
@@ -12,16 +12,15 @@
  │   See the License for the specific language governing permissions and       │
  │   limitations under the License.                                            │
  \*───────────────────────────────────────────────────────────────────────────*/
-'use strict';
```

Comment:
@hegdeashwin - actually, this is against the 3.0-alpha branch, so we only need to support node v6. I think we are good here. Thanks for the response.